client/CreatePostForm.tsx

```diff
@@ -2,10 +2,15 @@
 
 import { useRouter } from "next/navigation";
 import { trpc } from "~/client/trpcClient";
+import { rsc } from "~/server-rsc/trpc/client";
 import { Inputs } from "~/shared/utils";
 
 export function CreatePostForm() {
   const addPost = trpc.post.add.useMutation();
+  // const a = rsc.health.healthz.use();
+
+  const data = rsc.health.healthz.use();
```

Comment:
Wat, this works?

Reply:
Yes, see my other comment

Reply:
Ahhhhhh, I missed that this was imported from the `/client`....

I think this isn't great as you'll have to keep track of which one you're using 🤔

Reply:
That was a mistake (auto import). `server-rsc/trpc` should work too.